internal/provider-config/config.go

```diff
@@ -569,10 +569,10 @@ func DetermineLXDAddress(protocol string, address string) (string, error) {
 				port = "443"
 			}
 
-			address = fmt.Sprintf("%s:%s", address, port)
+			url.Host = fmt.Sprintf("%s:%s", url.Hostname(), port)
```

Comment:
Since we are dealing with strings I'd avoid the call to `Sprintf`:
```suggestion
			url.Host = url.Hostname() + ":" + port
```
But it's not performance critical ;)